packages/office-ui-fabric-react/src/components/Label/LabelCodepenButton.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { CommandButton } from '../..';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+
+const jsContent = require('!raw-loader!office-ui-fabric-react/src/components/Label/codepen_examples/Label.Basic.CodepenExample.txt') as string;
+const htmlContent = `<script src="//unpkg.com/office-ui-fabric-react/dist/office-ui-fabric-react.min.js"></script>
+<div id=\'content\'></div>`;
+
+const headContent = `<script type="text/javascript" src="https://unpkg.com/react@16/umd/react.development.js"></script><script type="text/javascript" src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>`;
+
+const valueData = {
+  title: 'Label Example Pen',
+  html: htmlContent,
+  head: headContent,
+  js: jsContent,
+  js_pre_processor: 'typescript'
+};
+
+// reformat the JSON string to take out the quotes so it'll work with the Codepen API
+const JSONstring = JSON.stringify(valueData)
+  .replace(/"/g, '&quot;')
+  .replace(/'/g, '&apos;');
+
+export const LabelCodepenButton = () => (
+  <div>
+    <form action="https://codepen.io/pen/define" method="POST" target="_blank">
```

Comment:
You can abstract this in a component which can be used in the `ExampleCard`. Think about what your inputs are, and then make sure the page provides it.

Reply:
I pushed changes that move all the boilerplate into ExampleCard to reduce the need for duplication, would you like me to abstract it away from ExampleCard into a separate component as well (I pushed before seeing this comment)?